weather_mv/loader_pipeline/ee.py

```diff
@@ -230,6 +231,8 @@ def add_parser_arguments(cls, subparser: argparse.ArgumentParser):
         subparser.add_argument('--ee_asset', type=str, required=True, default=None,
                                help='The asset folder path in earth engine project where the tiff image files'
                                ' will be pushed.')
+        subparser.add_argument('--ee_asset_type', type=str, default='IMAGE',
```

Comment:
Let's use choices instead of a string. 

https://docs.python.org/3/library/argparse.html#choices